library/src/utils/fake-url.ts

```diff
@@ -0,0 +1,52 @@
+import { SWARM_SESSION_ID_KEY } from './swarm-session-id'
+
+/** API endpoints that the extension can serve out */
+class FakeUrl {
```

Comment:
this is code duplication since it is available in `src/utils`. can you import it from one source?